pkg/logger/logger.go

```diff
@@ -14,12 +15,14 @@ Simplistic logger for audius-d.
 */
 
 var (
-	stdoutLogger *log.Logger
-	stderrLogger *slog.Logger
+	fileLogger  *slog.Logger
+	cliLogger   *slog.Logger
+	cliLevelVar = slog.LevelVar{}
+	logFilepath = filepath.Join(os.TempDir(), "audius-d.log")
```

Comment:
do we need these anymore? seems confusing to have this logger and also using the fmt ones

Reply:
I have the cliLogger and the fileLogger both used at the same time because the cliLogger won't include extra attributes like timestamp and whatnot and will be configured to be at INFO level by default, whereas the fileLogger will get everything